pennylane/devices/tests/conftest.py

```diff
@@ -31,20 +31,19 @@
 # Tolerance for non-analytic tests
 TOL_STOCHASTIC = 0.05
 # Number of shots to call the devices with
-N_SHOTS = 10000
+N_SHOTS = 1e6

```

Comment:
We could probably leave this at 10000, but the refactor uncovered some bugs that benefit from more shots.

Reply:
Does the increase in shots size here have a significant effect on the test runtime? If it is slower, could we just bump up the stochastic tolerance?

Reply:
It is a tricky balance...the stochastic tolerance is already awfully high. I think it should be fine...